src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/VariantAnnotatorEngine.java

```diff
@@ -216,6 +350,23 @@ private GenotypesContext annotateGenotypes(final ReferenceContext ref,
         return genotypes;
     }
 
+    /**
+     * Method which checks if a key is a raw key of the requested reducible annotations
+     * @param key
+     * @return
+     */
+    public boolean isRequestedReducibleRawKey(String key) {
+        if (reducableKeys == null) {
+            reducableKeys = new HashSet<>();
+            for (InfoFieldAnnotation annot : infoAnnotations) {
+                if (annot instanceof ReducibleAnnotation) {
+                    reducableKeys.add(((ReducibleAnnotation) annot).getRawKeyName());
```

Comment:
I guess I originally had it in my head that it would be an expensive operation for something that we won't use in the majority of cases, but its not really so i'll move it.